tests/integ/modin/sql_counter.py

```diff
@@ -364,6 +365,12 @@ def sql_count_checker(
         high_count_reason=high_count_reason,
     )
 
+    # Bypassing sql counter since
+    #   1. it is an unnecessary metric for tests running in stored proc
```

Comment:
i think you can even move this above, at the top of this call. Have you manually verified that works ?

Reply:
Yes, I tested this with integ test `tests/integ/modin/frame/test_head_tail.py` which ran successfully. I'll update the Snowfort POC with this test after this change